homeassistant/components/ozw/light.py

```diff
@@ -71,6 +68,8 @@ def __init__(self, values):
         self._white = None
         self._ct = None
         self._supported_features = SUPPORT_BRIGHTNESS
+        self._min_mireds = 154
```

Comment:
This is an internal instance attribute (`self._min_mireds`). It can't be overridden by customize. The property `min_mireds` will set a state attribute that can be overridden. But these are different things in the code. We're also not using the property `min_mireds` in the code, but the internal attribute.

Reply:
@MartinHjelmare basically in the long run this modification will allow for users to just customize the mireds for their light, if they're not in the 'default' range of 2700-6500k, no having to mod the code and less code maintenance/workarounds per light bulb required.

Reply:
I'm not following.

Reply:
Ah, I think I see what you're saying.  Should I swap the `self._min_mireds` for `self.min_mireds` in the code itself (other than the inital `__init__`)?